pom.xml

```diff
@@ -135,6 +135,56 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+              <groupId>org.apache.maven.plugins</groupId>
+              <artifactId>maven-dependency-plugin</artifactId>
+              <version>3.3.0</version>
+              <executions>
+                <execution>
+                  <id>analyze</id>
+                  <goals>
+                    <goal>analyze-only</goal>
+                  </goals>
+                  <configuration>
+                    <failOnWarning>true</failOnWarning>
+                    <ignoreNonCompile>true</ignoreNonCompile>
+                    <ignoredUnusedDeclaredDependencies>
+                      <ignoredUnusedDeclaredDependency>com.fasterxml.woodstox:woodstox-core</ignoredUnusedDeclaredDependency>
+                      <ignoredUnusedDeclaredDependency>commons-io:commons-io</ignoredUnusedDeclaredDependency>
+                      <ignoredUnusedDeclaredDependency>commons-logging:commons-logging</ignoredUnusedDeclaredDependency>
+                    </ignoredUnusedDeclaredDependencies>
```

Comment:
Can this section be removed and the issues fixed by removing the declarations of these dependencies from pom.xml?

Reply:
if we remove this section, we will get declared but not used error, though they are used by other dependent packages

Reply:
What do you mean by other dependent packages?

https://jlbp.dev/JLBP-22 dependencies should only be specified when project source files use them.

Reply:
updated